docs/tutorials/from-tabular-data-to-binned-data.ipynb

```diff
@@ -163,10 +163,28 @@
   },
   {
    "cell_type": "markdown",
-   "id": "e5619166-bc29-4e5c-b486-4e3f651ce625",
+   "id": "68928368-9995-421a-85a9-289ba64ac49e",
    "metadata": {},
    "source": [
-    "Alternatively, we can use [sc.bin](https://scipp.github.io/generated/functions/scipp.bin.html), which keeps the underlying events and their metadata:"
+    "Instead of specifying a specific bin *count* we may also specify a bin *size, given by a 0-D variable:"
```

Comment:
'specifying a specific' sounds a bit odd. You can leave out 'specific'.

And missing a * after size.

Reply:
```suggestion
    "Instead of specifying bin *count* we may also specify a bin *size*, given by a 0-D variable:"
```